theorems/T000520.md

```diff
@@ -0,0 +1,9 @@
+---
+uid: T000520
+if:
+  P000112: true
+then:
+  P000191: true
+---
+
+The topology contains a coarser topology which is {P53}, and {P53} implies {P191}.
```

Comment:
```suggestion
For each point, consider the collection of balls of radius $1/n$ from the coarser {P53} topology.
```

Might make the argument a bit more direct (don't have to convince yourself that P191 holds for all finer topologies).